### ISSUE TITLE
NTR: Liver biopsy: glyoxylate reductase activity low (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9159]
New term request Liver biopsy: glyoxylate reductase activity low (elthat_100628031103): GR, hepatic

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000638
- label: Reduced hepatic glyoxylate reductase activity
- parents: [Abnormal liver enzyme activity or concentration (HP:0034685)](https://hpo.jax.org/app/browse/term/HP:0034685)
- pmid: PMID:12185464
- def: Activity of glyoxylate reductase/hydroxypyruvate reductase (GRHPR; EC 1.1.1.79) in liver below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9159
- synonyms: Reduced hepatic hydroxypyruvate reductase activity(NONE)
